force-app/main/default/lwc/geTemplateBuilderFormField/geTemplateBuilderFormField.js

```diff
@@ -29,6 +29,17 @@ export default class geTemplateBuilderFormField extends LightningElement {
     hasPermission = hasViewSetup;
     shouldRender = true;
 
+    @api
```

Comment:
Is it correct that the component hierarchy requires that these methods be api enabled?